cmd/showcommit.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/qri-io/ioes"
+	"github.com/qri-io/qri/lib"
+	"github.com/spf13/cobra"
+)
+
+// NewShowCommitCommand creates a new `qri search` command that searches for datasets
+func NewShowCommitCommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
+	o := &ShowCommitOptions{IOStreams: ioStreams}
+	cmd := &cobra.Command{
+		Use:   "showcommit",
```

Comment:
Documentation of this this command should continue the work @Mr0grog landed in #1182. This should be `showcommit DATASET`